gammapy/catalog/fermi.py

```diff
@@ -340,12 +345,12 @@ def spectrum(self):
 
     def _get_flux_values(self, prefix='Flux', unit='cm-2 s-1'):
         if prefix not in ['Flux', 'Unc_Flux', 'nuFnu']:
-            raise ValueError("Must be one of the following: 'Flux', 'Unc_Flux', 'nuFnu'")
+            raise ValueError(
+                "Must be one of the following: 'Flux', 'Unc_Flux', 'nuFnu'")
```

Comment:
PyCharm by default fixes some stuff (like whitespace, ...) but leaves line breaks alone.
If you can configure your editor that way, and make pull requests that don't have stuff in the diff with unrelated changes (and in this case even changed i don't like), that would be good.

But it's not a big deal, we can still merge if you don't want to revert here now.